spk/jackett/Makefile

```diff
@@ -30,23 +31,31 @@ UNSUPPORTED_ARCHS = $(PPC_ARCHS)
 
 # Mono build for unsupported .NET archs
 include ../../mk/spksrc.archs.mk
-ifeq ($(findstring $(ARCH),$(ARMv5_ARCHS) $(i686_ARCHS) $(ARMv7L_ARCHS)),$(ARCH))
+include ../../mk/spksrc.common.mk
```

Comment:
From what I can tell the Jackett builds for both mono and dotnet packages. Given the compatibility issues with ARMv7 archs I did some logic to check for ARMv7 on DSM 6.x and build for mono but if on DSM 7.x build for dotnet unless it's an armada370. For this I needed commands from both .mk files.

Now I could simplify things and just build all the ARMv7 archs as mono and don't consider any of the nuances. This would only use the archs.mk. Would that be a preferred approach?